src/main/resources/templates/search-result.html

```diff
@@ -22,11 +22,11 @@ <h4>Search Result</h1>
 			<tr />
 		<thead />
 		<tbody>
-<!-- 		<tr th:each="theFaculty : ${faculties}"> -->
+		<tr>
 		<td th:Text="${carSearch.carMake}"/>
 		<td th:Text="${carSearch.carModel}"/>
 		<td th:Text="${carSearch.year}"/>
-		
+		<tr>
```

Comment:
This html was functional, but was throwing warnings in the Eclipse IDE. As it looks like you will only need the one row for now, I just made a single static tr element to remove the warning.